src/main/webapp/js/instructorFeedbackEdit/questionRubric.js

```diff
@@ -233,9 +227,7 @@ function highlightRubricRow(index, questionNum, highlight) {
 }
 
 function highlightRubricCol(index, questionNum, highlight) {
-    var idSuffix = getQuestionIdSuffix(questionNum);
-    
-    var $rubricCol = $('.rubricCol' + idSuffix + '-' + index);
+    var $rubricCol = $('.rubricCol-' + questionNum + '-' + index);
```

Comment:
@chowyb You mind creating the issue since you've compared across the question types?